app/server/__init__.py

```diff
@@ -18,6 +18,8 @@
 import sys
 import os
 
+from server.utils.phone import MessageProcessor
```

Comment:
should it still be in utils like this if we're using it within server/init?

does seem cleaner overall to declare it here with the various clients though, so we're not directly calling the clients in the util. if we want to ever extract message processor into a service, should be very easy

Reply:
> should it still be in utils like this if we're using it within server/init?

I mean we _could_ create a new module for it, but honestly this seems fine to me.